### PR TITLE
Update doxygen equations for MocoControlTrackingGoal and MocoSumSquaredStateGoal

### PR DESCRIPTION
**Pull request by chrisdembia**
Part of #333 

### Brief summary of changes

This PR fixes an error with the Doxygen equations for MocoControlTrackingGoal and adds equations for MocoSumSquaredStateGoal.

### CHANGELOG.md (choose one)

- [x] no need to update because...minor.

---
This change is [Reviewable](https://reviewable.io/reviews/opensim-org/opensim-moco/598)

### CODE REVIEW

**Comment by carmichaelong** [comment]
@chrisdembia Does the doxygen for any PR get posted somewhere automatically? No worries if not.

**Comment by chrisdembia** [comment]
Nope :(

File: Moco/Moco/MocoGoal/MocoSumSquaredStateGoal.h [hunk 1]
```diff
@@ -25,9 +25,21 @@ namespace OpenSim {
...
-/// This class also allows you to select which states to minimize by using
-/// a regex pattern with `setPattern()` and to provide weights for each
-/// state through `setWeightSet()` and `setWeightForState()`.
+///
+/// This goal is computed as follows:
+///
+/// \f[
+/// \int_{t_i}^{t_f} \sum_{s \in S} w_s y_s(t)^2 ~dt
+/// \f]
+/// We use the following notation:
```
Comment (carmichaelong) [inline Moco/Moco/MocoGoal/MocoSumSquaredStateGoal.h]: I think it's clear as is, or for completeness could add what `t_i` and `t_f` represent

File: Moco/Moco/MocoGoal/MocoContactTrackingGoal.h [hunk 2]
```diff
@@ -60,7 +60,8 @@ class OSIMMOCO_API MocoContactTrackingGoalGroup : public Object {
 ///
 /// \f[
 /// \frac{1}{mg} \int_{t_i}^{t_f}
-///         \sum_{g \in G} \|\mathrm{proj}_{\hat{n}}(F_{m,g} - F_{e,g})\|^2 ~dt
+///         \sum_{j \in G}
+///             \|\mathrm{proj}_{\hat{n}}(\vec{F}_{m,j} - \vec{F}_{e,j})\|^2 ~dt
 /// \f]
 /// We use the following notation:
 /// - \f$ mg \f$: the total weight of the system.
```
Comment (carmichaelong) [inline Moco/Moco/MocoGoal/MocoContactTrackingGoal.h]: Is `mg` only used below? Could defer definition to be down there (this is fine either way to me, especially since this isn't technically part of the PR)

File: Moco/Moco/MocoGoal/MocoContactTrackingGoal.h [hunk 3]
```diff
@@ -69,13 +70,13 @@ class OSIMMOCO_API MocoContactTrackingGoalGroup : public Object {
...
+/// - \f$ \vec{F}_{m,j} \f$ the sum of the contact forces in group \f$ j \f$,
 ///     expressed in ground.
-/// - \f$ F_{e,g} \f$ the experimental contact force for group \f$ g \f$,
+/// - \f$ \vec{F}_{e,j} \f$ the experimental contact force for group \f$ j \f$,
 ///     expressed in ground.
 ///
-/// If the model's gravity $g$ is 0, then we normalize by the total mass of the
-/// system instead of the total weight.
+/// If the model's gravity \f$ g \f$ is 0, then we normalize by the total mass
+/// of the system instead of the total weight.
```
Comment (carmichaelong) [inline Moco/Moco/MocoGoal/MocoContactTrackingGoal.h]: Again not part of the PR (and could be ignored or deferred to another PR), but this statement was a little confusing to me with "instead of" because the equation above does not show any normalization.

File: Moco/Moco/MocoGoal/MocoSumSquaredStateGoal.h (hunk 1 above)
Comment (chrisdembia) [reply Moco/Moco/MocoGoal/MocoSumSquaredStateGoal.h]: Addressed.

File: Moco/Moco/MocoGoal/MocoContactTrackingGoal.h (hunk 2 above)
Comment (chrisdembia) [reply Moco/Moco/MocoGoal/MocoContactTrackingGoal.h]: Addressed.

File: Moco/Moco/MocoGoal/MocoContactTrackingGoal.h (hunk 3 above)
Comment (chrisdembia) [reply Moco/Moco/MocoGoal/MocoContactTrackingGoal.h]: Addressed.

**Comment by chrisdembia** [comment]
Thanks @carmichaelong . Ready for review again.

**Comment by carmichaelong** [comment]
LGTM

**Comment by chrisdembia** [comment]
Thanks @carmichaelong :)

**Comment by chrisdembia** [comment]
Thanks for the reviews. I'll merge when tests pass (or feel free to merge if you catch the tests passing before me).

**Comment by chrisdembia** [comment]
Thanks :)
